chia/wallet/trading/offer.py

```diff
@@ -156,25 +156,35 @@ def get_offered_coins(self) -> Dict[Optional[bytes32], List[Coin]]:
 
             parent_puzzle: UncurriedPuzzle = uncurry_puzzle(parent_spend.puzzle_reveal.to_program())
             parent_solution: Program = parent_spend.solution.to_program()
-            additions: List[Coin] = [a for a in parent_spend.additions() if a not in self.bundle.removals()]
+            additions: List[Coin] = parent_spend.additions()
```

Comment:
I had to change this method because it was a little too heuristic and was picking up the wrong coin as "offered".  We now filter the additions to just unspent coins at the end of the function.